core/src/main/java/io/grpc/ClientCallImpl.java

```diff
@@ -85,27 +108,21 @@
     ClientTransport get();
   }
 
+  private enum State {INIT, STARTED, HALF_CLOSED, CANCELED}
```

Comment:
Elsewhere (like the canonical status codes) we are using the spelling cancelled, not canceled. It might be a good idea to use the double ell to have a consistent spelling.